pkg/segment/results/mresults/tests/metricsresults_test.go

```diff
@@ -404,3 +406,167 @@ func TestCalculateInterval(t *testing.T) {
 		}
 	}
 }
+
+func Test_GetResults_GTR(t *testing.T) {
+	test_GetResults_Ops(t,
+		map[uint32]float64{
+			0:     45,
+			10800: 1045,
+		},
+		map[uint32]float64{},
+		[]structs.QueryArithmetic{
+			{
+				LHS:        1,
+				ConstantOp: true,
+				Operation:  utils.LetGreaterThan,
+				Constant:   1045,
+			},
+		})
+}
+
+func Test_GetResults_GTE(t *testing.T) {
+	test_GetResults_Ops(t,
+		map[uint32]float64{
+			0:     45,
+			10800: 1045,
+		},
+		map[uint32]float64{
+			10800: 1045,
+		},
+		[]structs.QueryArithmetic{
+			{
+				LHS:        1,
+				ConstantOp: true,
+				Operation:  utils.LetGreaterThanOrEqualTo,
+				Constant:   100,
+			},
+		})
+}
+
+func Test_GetResults_LSS(t *testing.T) {
```

Comment:
nit: what does `LSS` mean? Maybe rename this to clarify it's testing "less than"